docs/class-definitions.md

```diff
@@ -74,3 +83,27 @@ This could be the local client or other remote clients which are connected to th
 | isConnected | bool                              |
 | profileData | Object                            |
 | lastEvent   | {name: string, timestamp: number} |
```

Comment:
I think we need to add a `location` field here now?

Reply:
Fixed https://github.com/ably-labs/spaces/pull/41/commits/ec228813f179b7068bb7b649a79fcaa81465fcc0